code/game/gamemodes/objective.dm

```diff
@@ -352,13 +352,7 @@ GLOBAL_LIST_EMPTY(objectives)
 	var/hijack_speed_override = 1
 
 /datum/objective/hijack/check_completion() // Requires all owners to escape.
-	if(SSshuttle.emergency.mode != SHUTTLE_ENDGAME)
-		return FALSE
-	var/list/datum/mind/owners = get_owners()
-	for(var/datum/mind/M in owners)
-		if(!considered_alive(M) || !SSshuttle.emergency.shuttle_areas[get_area(M.current)])
-			return FALSE
-	return SSshuttle.emergency.is_hijacked()
+	return FALSE // PLACEHOLDER
```

Comment:
Hmm.

Reply:
It's just objective code, and it currently doesn't work anyways. Would rather not *also* remove all objective code in this PR